sdk/src/mcm_dp.c

```diff
@@ -146,6 +146,10 @@ static void parse_memif_param(mcm_conn_param* request, memif_socket_args_t* memi
             log_error("Invalid audio parameters.");
         break;
 
+    case PAYLOAD_TYPE_RDMA_VIDEO:
+        request->payload_args.rdma_args.transfer_size = (size_t)request->payload_args.video_args.width * request->payload_args.video_args.height * 4;
```

Comment:
I don't understand what is going on here. I believe, `parse_memif_param()` is called only when pure memif connection is created between apps, but we need to calculate transfer_size when RDMA connection is set up via media_proxy.

We could calculate it in media proxy itself, but probably better idea would be to calculate it before calling `mcm_create_connection_proxy()` 

I would also encourage you to calculate transfer_size with consideration of the video format, rather than just multiplying the number of pixels by 4.